src/advisory.jl

```diff
@@ -320,13 +328,17 @@ function to_osv_dict(a::Advisory)
 end
 # Package vulnerabilities are the one thing we store quite differently:
 function to_osv_dict(vuln::PackageVulnerability)
-    return Dict{String, Any}(
+    osv_dict = Dict{String, Any}(
         "package" => OrderedDict(
             "ecosystem" => "Julia",
             "name" => vuln.pkg,
-            # TODO: "purl" => purl(vuln.pkg)
+            "purl" => purl(vuln)
         ),
         "ranges" => [OrderedDict("type"=>"SEMVER", "events"=>osv_events(vuln.ranges))],
         # TODO: "versions" => registered_versions_within_the_ranges(vuln.pkg, vuln.ranges)
     )
+    url = purl(vuln)
+    if !isnothing(url)
```

Comment:
And then we don't need to worry about doing any sorts of fixups to the ["package"]["purl"] field.